src/lib/space-map-image.ts

```diff
@@ -0,0 +1,29 @@
+import type { Space } from "~/types/Space";
+
+/**
+ * Create a static map of building map with space marker
+ *
+ * @return image of building map with space marker
+ */
+export default function (space: Space, mapboxToken: string) {
+  const { bounds } = space.building;
+  // Use invisible markers at the bounds of the building
+  // and set zoom to `auto` so map extends exactly to building
+  const invisibleMarkerUrl = encodeURIComponent(
+    "https://5d4b289a581fb6255e317d9d--assets-jasper.netlify.com/no-marker.png"
+  );
+  const width = 800;
+  const height = width;
+  const primaryColor = "#8B2BA7";
+  const url = [
```

Comment:
Should we use the native [`URL` API](https://developer.mozilla.org/en-US/docs/Web/API/URL) for this?

Reply:
It's still some concatenations of segments with a comma separator inside. I don't think we'll have a lot of profit if we make it kinda more semantic